### ISSUE TITLE
Optimal way to store and read in MSD

### ISSUE THREAD

**Issue by achafetz** [USAID-OHA-SI/gophr#9]
We currently store and use rds format, via `read_msd(store = TRUE)`. 

`read_msd()` reads in rds files using `readr::read_rds` and zip/txt with `vroom::vroom()`.

Looking at the size and read in times, it may be better just to keep the MSDs in their zip files.

![image](https://user-images.githubusercontent.com/8933069/159732071-c9620a33-4b0d-44a0-8500-9a2626ae4bae.png)
*tested on the FY22Q2i MSD

**Comment by baboyma** [comment]
Agreed. It might still be worth changing the function parameter 'save_rds` to `save_as` with these file types as options. Thoughts?

**Comment by achafetz** [comment]
I like it. So rather than T/F, we include something like `save_as` or `output_format`. It looks like the optimal way to to leave as is and not output (default would be `NULL`/`missing`), but would give the user the ability to define the out as either - `feather` or `rds`. Any other useful output format I'm not thinking about? Not sure there is value in outputting as a txt or csv

**Comment by baboyma** [comment]
Humm, now you are opening the door to wildness. How about .hyper or .db (for sqlite files)?

csv and txt for good outputs used for importing data into RDBMS. 

You can with the small subset and expand as needed.
